src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -594,6 +595,13 @@ private function getErrorMessages(Exception $e)
                 'Gift message not valid',
                 'Admin.Notifications.Error'
             ),
+            MinimalQuantityException::class => $this->trans(
+                'You must add a minimum quantity of %d',
+                'Admin.Orderscustomers.Notification',
```

Comment:
Suggestion, we have `You must add %d minimum quantity` in `Admin.Orderscustomers.Notification` that already exist

Reply:
The one you used already exists, I don't know which one should be used 🤷‍♂️

Reply:
I use the 1768 error message. I wait for @LouiseBonnard review ;)

Reply:
It's @Julievrz who validates wording now 😉

Reply:
I think " You must add a minimum quantity of d%" sounds better. Do you confirm @LouiseBonnard ?

Reply:
Ok. Sorry @jolelievre. The wording is validated ;)